packages/@okta/vuepress-theme-prose/assets/css/components/_selectorTile.scss

```diff
@@ -0,0 +1,94 @@
+.selector-tile{
+  border: 2px solid #1F3B54;
+  border-radius: 15px;
+  color: #ffffff;
+  width: 100%;
+  background: linear-gradient(rgba(22,39,47,1) 66%, rgba(26,49,66,1) 84%);
+  height: 375px;
+  box-sizing: border-box;
+  overflow: hidden;
+  box-shadow: 2px 33px 30px -26px rgba(18,30,36,1);
+  @include media("<=bootstrap-xl"){
+    margin-bottom: 30px;
+  }
+  @include media("<=bootstrap-md"){
+    height: auto;
+  }
+
+  &--header{
+    font-size: 18px;
+    padding: 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    color: #DFEAF1;
+    &:visited,&:active{
+        color: #DFEAF1;
+    }
+    >i,span{
+      display: block;
+    }
+    >i{
+      color: #bac1c6;
+    }
+    &:hover{
+      color: #ffffff;
+      >i{
+        color: #ffffff;
+      }
+    }
+  }
+  &--data{
+    &--item{
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      padding: 10px 20px;
+      background-color: #1a3142;
+      // border-image-slice: 1;
```

Comment:
Could you create a Jira ticket to track the issue that was caused by this instead and remove this ?